### PR TITLE
methods for pr and issue labels

### LINKED ISSUE #171: Implement GitLab methods for issue labels
```python
    def get_issue_labels(self, issue_id: int) -> List:
        raise NotImplementedError()

    def add_issue_labels(self, issue_id, labels) -> None:
        raise NotImplementedError()
```

- https://python-gitlab.readthedocs.io/en/stable/gl_objects/issues.html
- https://docs.gitlab.com/ce/api/issues.html

### PR DESCRIPTION
**Pull request by lbarcziova**
Fixes #171, #174

### CODE REVIEW

File: ogr/services/gitlab.py [hunk 1]
```diff
@@ -341,10 +353,22 @@ def pr_merge(self, pr_id: int) -> "PullRequest":
         return self._pr_from_gitlab_object(pr)
 
     def get_pr_labels(self, pr_id: int) -> List:
```
Comment (lachmanfrantisek) [inline ogr/services/gitlab.py]: Maybe a more specific type?

```suggestion
    def get_pr_labels(self, pr_id: int) -> List[str]:
```

**Comment by lbarcziova** [comment]
Yes, I have generated them again, but it seems like there is some problem with the responses.

**Comment by lachmanfrantisek** [comment]
> Yes, I have generated them again, but it seems like there is some problem with the responses.

I'll take a look.

**Comment by lachmanfrantisek** [comment]
Just rebase and we can merge. (Sorry for another one.)

**Pull request merged**